docs/guides/mbed_commissioning.md

```diff
@@ -0,0 +1,184 @@
+payload is encoded within a QR code and is printed to the UART console.
+
+    	[SVR] Copy/paste the below URL in a browser to see the QR Code:
+        https://dhrishi.github.io/connectedhomeip/qrcode.html?data=CH%3AI34DV%2A-00%200C9SS0
+        Open the URL in a web browser to have the commissioning QR code generated.
+
+
+  Reset of the device.
+
+  Find a message similar to the following one in the application logs:
+
+  Open URL from the console so QR code can be visible in a web browser.  
+   ![CHIPTool qr screen](../images/qr.png)
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/ARMmbed/mbed-os/master/logo.png" alt="ARM Mbed-OS logo"/>
+</p>
+<a name="commisionig_android"></a>
+## Device commisioning  for Android
+
+To commission CHIP device onto the  network created complete the following steps:
+
+ Enable Bluetooth and Location services on your smartphone.
+
+ Connect the smartphone to the Wi-Fi Network
+
+ Open the CHIPTool application on your smartphone.
+
+ Tap the PROVISION CHIP DEVICE WITH WIFI button and scan the commissioning QR code.
+
+  You will see a few pop-up messages appear as the commissioning progresses. Finally, the network settings screen appears.
+
+  ![CHIPTool scanning screen](../images/scanning.jpg)
+  ![CHIPTool pairing screen](../images/pairing.jpg)
+  ![CHIPTool net screen](../images/wifi_network.jpg)
+  In the new screen tap the Save Network button to send a Thread provisioning message to the accessory device.
+
+Screen with WIFI password  will disappear and app returns to main screen
+ 
+<a name="sending_app_data_android"></a>
+## Sending Application packets for Android
+
+After the accessory device is successfully commissioned to the network, it is possible to communicate with it using IP. CHIP uses Zigbee Cluster Library (ZCL) protocol which defines common means for applications to communicate.
+
+From main Chip tool screen select "LIGHT ON/OFF & LEVEL CLUSTER"
+
+   ![CHIPTool app screen](../images/app.jpg)
+
+
+To control the light dimming, please follow the below steps.
+
+
+Tap either the ON or the OFF button to toggle between  min and max brightness.
+Use the slider to modify it between 0-255
+
+If **Lighting LED** is connected than brightness change can be observed.
+
+
+
+<a name="tool_linux"></a>
+## ChipTool Linux
+
+
+Building 
+
+	cd examples/chip-tool
+	git submodule update --init
+	source third_party/connectedhomeip/scripts/activate.sh
+	gn gen out/debug
+	ninja -C out/debug
+
+After the application is built, it can be found in the build directory as out/debug/chip-tool
+
+
+<a name="commisionig_linux"></a>
+## Device commisioning Linux
+
+In order to send commands to a device, it must be paired with the client.
+
+To initiate a client pairing request to a device, run the built executable and pass it the discriminator and pairing code of the remote device.
+
+The command below uses the default values hard-coded:
+
+$ chip-tool pairing ble 20202021 3840
+
```

Comment:
Can you add commands to send ZCL messages to the device after commissioning ?

Reply:
Done